lm75bd/lm75bd.c

```diff
@@ -29,6 +29,36 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
   
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  } 
+  uint8_t tempWriteBit = 0x00;
+ 
+  i2cSendTo(devAddr, &tempWriteBit,(uint16_t) 1);
+  uint8_t tempData[2];
+  i2cReceiveFrom(devAddr, tempData, (uint16_t) 2);
+
+  /* Concatenate the received data */
+  uint16_t rawData = (tempData[0] << 8) | tempData[1];
+
+  /* Determine if received temperature is positive or negative */
+  uint16_t posOrNeg = rawData & 0x8000;
+
+  /* Bitshift right 5 because the 5 LSB are useless so delete */
+  rawData = rawData >> 5;
+
+  if (posOrNeg == 0) /* Positive Temperature */{ 
+    *temp = (float)(rawData) * 0.125;
+  }
+
+  else /* Negative Temperature*/ {
+    /* Convert to two's complement */
+    uint16_t flipLeadingBits = 0xF800;
```

Comment:
Since this is a constant, declare it with `const`

Reply:
this can also just be made into a macro